lms/envs/aws.py

```diff
@@ -557,6 +557,7 @@
 
 ##### CDN EXPERIMENT/MONITORING FLAGS #####
 CDN_VIDEO_URLS = ENV_TOKENS.get('CDN_VIDEO_URLS', CDN_VIDEO_URLS)
+ONLOAD_BEACON_SAMPLE_RATE = ENV_TOKENS.get('ONLOAD_BEACON_SAMPLE_RATE', ONLOAD_BEACON_SAMPLE_RATE)
```

Comment:
That's in FEATURES, which is loaded [earlier in the file](https://github.com/edx/edx-platform/blob/master/lms/envs/aws.py#L234).